code/game/objects/structures/crates_lockers/crates/secure.dm

```diff
@@ -38,11 +38,16 @@
 	explosion(src, heavy_impact_range = 1, light_impact_range = 5, flash_range = 5)
 	qdel(src)
 
+/datum/armor/crate_secure/weapons // return to emitters
+	melee = 100
+	bullet = 100
+
 /obj/structure/closet/crate/secure/weapon
-	desc = "A secure weapons crate."
+	desc = "A secure weapons crate. The specially alloyed material hardens in response to physical trauma."
```

Comment:
Since the armor is kind of snowflaked to level 100, i think it'd be best for the description to show it doesn't have a resistance to lasers, and especially heavy laser systems. gotta think of the newbies.

Reply:
What if it's like "The specially alloyed material hardens in response to physical trauma. It seems susceptible to being melted, though."